File: src/supabaseClient.js
import { createClient } from '@supabase/supabase-js';

// Your Supabase URL and API Key
const supabaseUrl = 'https://uypwmotuasqmhhtibylz.supabase.co'; // replace with your project URL
const supabaseKey = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InV5cHdtb3R1YXNxbWhodGlieWx6Iiwicm9sZSI6ImFub24iLCJpYXQiOjE3MzI5NjU4NDQsImV4cCI6MjA0ODU0MTg0NH0.j5lwZvl-p-MeGSIvAazZPTvqlrDKWgXa_XBpMPuIgLw'; // replace with your API key

const supabase = createClient(supabaseUrl, supabaseKey);

export default supabase;

File: ticketing_backend/supabase_client.py
from supabase import create_client, Client

# Your Supabase URL and Key from the Supabase project settings
url = "https://uypwmotuasqmhhtibylz.supabase.co"
key = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InV5cHdtb3R1YXNxbWhodGlieWx6Iiwicm9sZSI6ImFub24iLCJpYXQiOjE3MzI5NjU4NDQsImV4cCI6MjA0ODU0MTg0NH0.j5lwZvl-p-MeGSIvAazZPTvqlrDKWgXa_XBpMPuIgLw"
supabase: Client = create_client(url, key)
